AnimatedComponents/src/components/floating-button/circular-floating-button/CircularFloatingButton.tsx

```diff
@@ -32,42 +32,44 @@ const CircularFloatingButton = (props: FloatingButtonProps) => {
       : withTiming(0, { duration: animationTransitionDuration }),
   )
   const { width } = Dimensions.get('window')
-  const FAB_SIZE = 54
-  const circleScale = Number((width / FAB_SIZE).toFixed(1))
-  const circleSize = circleScale * FAB_SIZE
-  const dist = circleSize / 2 - FAB_SIZE * 1.5
-  const radius = dist * 1.5
+  const iconSize = 54
+  const circleScale = Number((width / iconSize).toFixed(1))
+  const circleSize = circleScale * iconSize
+  const angularDistance = circleSize / 2 - iconSize * 1.5
```

Comment:
add comment to explain this , what distance we mean here.

Reply:
The angularDistance determines how far each icon is placed from the center of the circular layout. This all is done mathematically by some calculations.

Reply:
Add this to code as a comment
"angularDistance determines how far each icon is placed from the center of the circular layout"